test/runtests.jl

```diff
@@ -614,16 +614,16 @@ end
 @testset "EquiRect" begin
 
     θspan = deg2rad.((140,110))
-    ϕspan = deg2rad.((-10,90))
+    φspan = deg2rad.((-10,90))
     Cℓ = camb()
 
     @testset "Nside=$Nside" for Nside in Nsides
 
         local f0, f2, Cf0, Cf2
 
         # constructor doesnt error
-        @test (f0 = EquiRectMap(rand(Nside...); θspan, ϕspan)) isa EquiRectMap
-        @test (f2 = EquiRectQUMap(rand(Nside...), rand(Nside...); θspan, ϕspan)) isa EquiRectQUMap
+        @test (f0 = EquiRectMap(rand(Nside...); θspan, φspan)) isa EquiRectMap
```

Comment:
Btw, I'm happy keeping a constructor with passing in a manual `θ` / `φ` grid, but I think we should also have a very simple one where you pass in `θspan`, `φspan` and it assumes `:equiθ` to match the BK projection.